components/tidb_query/Cargo.toml

```diff
@@ -5,6 +5,18 @@ edition = "2018"
 publish = false
 description = "A query engine to run TiDB pushed down executors"
 
+[features]
+protobuf-codec = [
```

Comment:
Can you also give it a default feature?

Reply:
Please add the default feature for tidb_query.

Reply:
Why need it?

Reply:
Because `tidb_query` is also a crate in heavy development, we would like to let cargo work in there.

Reply:
You can use `cargo build -p tidb_query` or `make expression`
If we add default proto feature for `tidb_query`, it will cause compilation problems, like:
```
export PROST=1
make clippy

55  | /     pub fn generate_files(&self) {
56  | |         let mut cmd = Command::new(get_protoc());
57  | |         let desc_file = format!("{}/mod.desc", self.out_dir);
58  | |         for i in &self.includes {
...   |
101 | |         self.replace_read_unknown_fields();
102 | |     }
    | |_____^ duplicate definitions for `generate_files`
    | 
   ::: /home/koushiro/.cargo/git/checkouts/protobuf-build-174b08b051947a10/42e52b9/src/prost_impl.rs:5:5
    |
5   | /     pub fn generate_files(&self) {
6   | |         #[cfg(feature = "grpcio-prost-codec")]
7   | |         {
8   | |             grpcio_compiler::prost_codegen::compile_protos(
...   |
24  | |             .for_each(|path| WrapperGen::new(path, self.wrapper_opts).write());
25  | |     }
    | |_____- other definition for `generate_files`

```

Reply:
I can't reproduce this error in my local environment, so I pushed it here to let ci do the rest.

Reply:
It seems like a bug of Cargo that default feature is not possible to be disabled... 😢

Reply:
I guess yes, I was stuck with this problem for a long time :cry:

Reply:
I'm sorry to let you tweak them back and forth. This is my mistake that I think the giving tidb_query default feature can make `cargo check` simply runs in `component/tidb_query`. Can you revert it back to `5eaca98`? I'm very appreciated.

Reply:
Done